buildsystem/codecompliance/__main__.py

```diff
@@ -137,17 +140,49 @@ def main(args):
     else:
         check_files = None
 
+    auto_fixes = list()
+
     issues_count = 0
-    for title, text in find_all_issues(args, check_files):
+    for title, text, apply_fix in find_all_issues(args, check_files):
         issues_count += 1
         print("\x1b[33;1mWARNING\x1b[m {}: {}".format(title, text))
 
+        if apply_fix:
+            if args.fix:
+                print("        This will be fixed automatically.")
+                auto_fixes.append(apply_fix)
```

Comment:
If `args.fix` is `False`, no fixes ever get appended to `auto_fixes`. This does not seem to be your intention, as you are checking for `not args.fix and auto_fixes` below for printing a message about possible fixes. I'd just omit checking `args.fix here` and always add the fix to the list and write "can be fixed". This would also make pylint happy (too many branches).